src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -205,12 +207,17 @@ export const SelectRegistrationTypeField = () => {
     }
   };
 
+  const [searchValue, setSearchValue] = useState('');
+
+  const filterSubTypes = (subTypes: RegistrationTypeElement[]) => {
+    return subTypes.filter((subType) => subType.text.toLowerCase().includes(searchValue.toLowerCase()));
+  };
```

Comment:
Sparer en linje (eller to med litt flaks) 😅 
```suggestion
  const filterSubTypes = (subTypes: RegistrationTypeElement[]) => subTypes.filter((subType) =>subType.text.toLowerCase().includes(searchValue.toLowerCase()));
```

Reply:
tenker vi også burde bruke nye begrep i stedet for `subType`, siden det ikke lenger gir mening etter at vi endret type-velgeren. `type` tror jeg faktisk holder, evt `registrationType`. Lener mot `type`.. hva tror du?

Reply:
Tenker kanskje at `type` er ganske generelt, så heller egentlig mer mot `registrationType`.

Reply:
kjøper den!